libs/payments/paypal/src/lib/paypal.manager.ts

```diff
@@ -166,12 +177,31 @@ export class PayPalManager {
     const amountInCents = invoice.amount_due;
 
     if (amountInCents < this.stripeManager.getMinimumAmount(invoice.currency)) {
-      return await this.processZeroInvoice(invoice.id);
-    } else {
-      const customer = await this.stripeManager.fetchActiveCustomer(
-        invoice.customer
-      );
-      return await this.processNonZeroInvoice(customer, invoice);
+      await this.processZeroInvoice(invoice.id);
+      return;
+    }
+
+    const customer = await this.stripeManager.fetchActiveCustomer(
+      invoice.customer
+    );
+    await this.processNonZeroInvoice(customer, invoice);
```

Comment:
Went ahead and did this because the return values/typings were a little strange here -- not entirely related to the topic of this PR but polish.